recipes/pycryptodomex/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "pycryptodomex" %}
+{% set version = "3.4.7" %}
+{% set sha256= "52aa2e540d06d63636e4b5356957c520611e28a88386bad4d18980e4b00e0b5a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+  run:
+    - python
+    - gmp   # [not win]
+    - mpir  # [win]
```

Comment:
Should we be constraining these somehow?

Reply:
The project on github doesn't say anything about the version on these dependencies.
What do you think? Also: https://abi-laboratory.pro/tracker/timeline/gmp/

Reply:
Not sure. It's not technically linking to these libraries during the build. So it may not have the same issues/constraints that a typical library or executable have. Am ok with leaving these unpinned until we find out there are actual issues. Figured I'd ask if you had any thoughts on it first though. ;)